dist/README.md

```diff
@@ -0,0 +1,75 @@
+# 🌀 typhoon-ui
```

Comment:
Can you remove all the `dist` from the changes, since we don't want to merge them.